### ISSUE TITLE
Acceptance tests

### ISSUE THREAD

**Issue by cmitz** [csvalpha/sofia#131]
Vervolg op #101 .

Ik wil graag een paar acceptance tests om te zorgen dat de UI werkt zoals verwacht. Dat betekent het configureren van een headless browser op CI (in een nieuwe task) en het opzetten van Capybara o.i.d, in combinatie met Page Objects pattern

**Comment by lodewiges** [comment]
i dont think we are going to implement this anymore. I will close the pull request for now
